tests/integration/security/authorization_test.go

```diff
@@ -906,54 +831,114 @@ func TestAuthorization_TCP(t *testing.T) {
 					ExpectAllowed: expectAllowed,
 				}
 			}
+			ns := apps.Namespace1
+			ns2 := apps.Namespace2
+			src0 := apps.A.Match(echo.Namespace(ns.Name()))
```

Comment:
Can we name the `src0`, `src1`, ..., the same as its workload (e.g. `a`, `b`, ...), this will make it much easier to compare with the current test.